lib/experimental/class-wp-html-walker.php

```diff
@@ -137,7 +137,7 @@ private function parse_next_attribute() {
 
 		$this->current_tag_matched_attributes[ $attribute_name ] = new WP_HTML_Attribute_Match(
 			$attribute_name,
-			self::comparable( $attribute_value ),
+			$attribute_name === 'class' ? self::comparable( $attribute_value ) : '',
```

Comment:
I didn't mean to commit this without a comment, so just to explain – we only ever use the "class" attribute value so it doesn't make sense to store the other ones.